docs/tools/cli/service/connection-info.rst

```diff
@@ -0,0 +1,211 @@
+``avn service connection-info``
+==================================================
+
+Here you’ll find the full list of commands for ``avn service connection-info``.
+
+
+Retrieve connection information
+--------------------------------------------------------
+
+``avn service connection-info kafkacat``
+'''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''
+
+Retrieves the ``kcat`` :doc:`command </docs/products/kafka/howto/kcat>` necessary to connect to a certain Aiven for Apache Kafka® service.
+
+.. list-table::
+  :header-rows: 1
+  :align: left
+
+  * - Parameter
+    - Information
+  * - ``service_name``
+    - The name of the service
+  * - ``--route``
+    - The type of route to use to connect to the service. Possible values are ``dynamic``, ``privatelink`` and ``public``
+  * - ``--privatelink-connection-id``
+    - The Id of the privatelink to use
+  * - ``--kafka-authentication-method``
+    - The Aiven for Apache Kafka® authentication method. Possible values are ``certificate`` and ``sasl``
+  * - ``--username``
+    - The username used to connect if using ``sasl`` authentication method
+  * - ``--ca``
+    - The path to the CA certificate file
+  * - ``--client-cert``
+    - The path to the client certificate file
+  * - ``--client-key``
+    - The path to the client key file
+  * - ``--write``
+    - Save the certificate and key files if not existing
+  * - ``--overwrite``
+    - Save (or overwrite if already existing) the certificate and key files
+
+**Example:** Retrieve the ``kcat`` command to connect to an Aiven for Apache Kafka service named ``demo-kafka`` with SSL authentication (``certificate``), download the certificates necessary for the connection:
+
+::
+
+  avn service connection-info kafkacat demo-kafka --write
+
+An example of ``service connection-info kafkacat`` output:
+
+.. code:: text
+
+  kafkacat -b demo-kafka-dev-advocates.aivencloud.com:13041 -X security.protocol=SSL -X ssl.ca.location=ca.pem -X ssl.key.location=service.key -X ssl.certificate.location=service.crt
+
+.. Warning::
+
+  The command output uses the old ``kafkacat`` naming. To be able to execute ``kcat`` commands, replace ``kafkacat with ``kcat``.
```

Comment:
Can we please just fix this in the CLI tool?

Reply:
The last `kafkacat` is missing its closing literal/code formatting, it's rendering slightly funky.